src/plugins/telemetry/public/services/telemetry_sender.ts

```diff
@@ -53,13 +53,22 @@ export class TelemetrySender {
     return false;
   };
 
+  private delayRetrySend = () => {
+    return 60 * (1000 * Math.min(Math.pow(2, this.retryCount), 64)); // 120s, 240s, 480s, 960s, 1920s, 3840s, 3840s, 3840s
+  };
+
   private sendIfDue = async (): Promise<void> => {
-    if (this.isSending || !this.shouldSendReport()) {
+    if (!this.shouldSendReport()) {
       return;
     }
+    // optimistically update the report date and reset the retry counter for a new time report interval window
+    this.lastReported = `${Date.now()}`;
```

Comment:
We have a behavior change here where we're now updating the report date _before_ sending the data vs only after a successful send. We need to do this to ensure we're backing off the retry attempts.